pkg/image/registry/imagestream/etcd/etcd.go

```diff
@@ -63,13 +64,15 @@ func NewREST(optsGetter restoptions.Getter, defaultRegistry api.DefaultRegistry,
 	statusStore.Decorator = nil
 	statusStore.CreateStrategy = nil
 	statusStore.UpdateStrategy = statusStrategy
+	statusStore.DeleteStrategy = nil
```

Comment:
Can't have nil once we drop https://github.com/openshift/kubernetes/commit/0e612f0ab1227466f701991d03a627b5bba2ffe7